sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/CliSuite.scala

```diff
@@ -644,7 +644,8 @@ class CliSuite extends SparkFunSuite {
 
   test("SPARK-37694: delete [jar|file|archive] shall use spark sql processor") {
     runCliWithin(2.minute, errorResponses = Seq("ParseException"))(
-      "delete jar dummy.jar;" -> "Syntax error at or near 'jar': missing 'FROM'.(line 1, pos 7)")
+      "delete jar dummy.jar;" ->
+        "Syntax error at or near 'jar': missing 'FROM'. SQLSTATE: 42601(line 1, pos 7)")
```

Comment:
Looking at this, should we add a '.' after SQLSTATE like:
```suggestion
        "Syntax error at or near 'jar': missing 'FROM'. SQLSTATE: 42601. (line 1, pos 7)")
```

Reply:
It's not a sentence. But I agree with a space. I need to find where the (line, pos) is coming from. And prefix the space.